### ISSUE TITLE
Not building with electron-builder

### ISSUE THREAD

**Issue by arcovoltaico** [castlabs/electron-releases#191]
having :
```
    "electron": "github:castlabs/electron-releases#v33.0.0+wvcus",
    "electron-builder": "25.1.8",
```
when I run

`npm run build:prod && npx electron-builder build --mac`

results in :

```
  • packaging       platform=darwin arch=x64 electron=33.0.0+wvcus appOutDir=release/mac
  ⨯ cannot resolve https://github.com/electron/electron/releases/download/v33.0.0+wvcus/electron-v33.0.0+wvcus-darwin-x64.zip: status code 404
github.com/develar/app-builder/pkg/download.(*Downloader).follow
        /Volumes/data/Documents/app-builder/pkg/download/downloader.go:237
github.com/develar/app-builder/pkg/download.(*Downloader).DownloadNoRetry
```

**Comment by arcovoltaico** [comment]
ok, the installing example is a bit misleading, it should point to the latest version, v31.7.3+wvcus
not to a non-existent one  :-)

**Comment by arcovoltaico** [comment]
Also, I needed to add this to my `electron-builder.json`
```
  "electronDownload": {
      "mirror": "https://github.com/castlabs/electron-releases/releases/download/"
    }
```

**Comment by khwaaj** [comment]
Great that you got it working, just a couple of quick clarifications for you and others finding this.

* `v33.0.0+wvcus` is a valid version at the time of this writing. We update the README for each major version released, so it is supposed to always point to the first version in the latest stable release series (see https://github.com/castlabs/electron-releases/wiki#supported-versions). It is always prudent to double-check which version you actually want to use, e.g. the latest in the `v33` series is the just published `v33.1.0+wvcus`.
* The real issue here is the missing `electron-builder` mirror. This is mentioned as a prerequisite in our FAQ describing how to use ECS with `electron-builder` (see https://github.com/castlabs/electron-releases/wiki/FAQ#how-can-i-use-electron-builder-with-ecs).
